### ISSUE TITLE
Directory deletion in file manager does not delete it from file explorer of Windows

### ISSUE THREAD

**Issue by aricodeine** [Zettlr/Zettlr#4656]
### Description

Im currently using Windows 11 latest version.
Bug: When i create a new directory in Zettlr and delete it, it is only deleted from the view of file manager of Zettlr.

Expected: Deleting a directory from Zettlr should also delete it from the File Explorer of Windows.
![image](https://github.com/Zettlr/Zettlr/assets/88302441/cc88fa82-6e13-4313-8409-2172fc197fcf)
As shown in the above pic, the directory "Hello" is deleted from the file manager of Zettlr but it is not deleted from the system.

### Reproducing

I am a beginner and looking to get into open source contributions so I do not know how to fix this but am looking forward to fix this. I have no knowledge of JS and TS so I would love to team up with someone to fix this.

### Zettlr Version

Stable (most recent version)

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 11

**Comment by nathanlesage** [comment]
Would you mind checking the logs? Zettlr should take note if deleting a directory fails. You can find them with the debug mode enabled (advanced preferences) in the menu Develop → Open logs.

**Comment by aricodeine** [comment]
Sure.

![image](https://github.com/Zettlr/Zettlr/assets/88302441/c8d26c3d-4134-4dc9-9852-d39bdf20a45b)

Here as you can see in the logs, I had created the "hello" directory which i have deleted but its log didn't show up and the "hello" folder is still there in the File Explorer of Windows.
Now, if I delete the Folder from File Explorer, the 'unlinkDir' event log will be shown on the log viewer as shown in the image below:
![image](https://github.com/Zettlr/Zettlr/assets/88302441/68153c84-d8ab-4792-a2f3-6af3005fc54a)


So, in short, deleting the "hello" directory only deletes the directory view from the File manager of Zettlr but does not create the unlinkDir log, but it creates unlinkDir log only when I delete the same directory manually from the File Explorer of Windows

**Comment by lxwulf** [comment]
Yes, I can approve this behavior too, explained from @HJKLE on Fedora Linux 38 with GNOME desktop.

**Comment by pseandersson** [comment]
@nathanlesage I will start looking into this
